packages/field/src/FormatMixin.js

```diff
@@ -259,6 +259,11 @@ export const FormatMixin = dedupeMixin(
         }
       }
 
+      // TODO: refactor formatter/parser so this listener is not needed
+      _errorStateHandler() {
```

Comment:
Maybe give it a more specific name like `_reformatOnErrorStateChanged` to prevent potential naming collisions on the prototype (since it's quite a generic name)?

In the comment, maybe add:
```
// Current flow works like this: setting of modelValue triggers observers for:
// - computation of 'formattedValue'
// - validation (and thus computation of 'errorState')
// Observers will be handled in above order. Since computation of formattedValue depends on 'errorState' 
// (erroneous inputs should not be formatted), the formatter is handled with a not yet updated errorState.
// 
// TODO: rewrite to a flow handling observers in this order: modelValue -> errorState(validation) -> formattedValue
// Important: currently, a situation where a value changes from valid to invalid will still lead to formatting when not 
// required. 
// Since the flow above involves two mixins (FormatMixin and ValidateMixin) and ValidateMixin should have no
// knowledge about FormatMixin, think of a proper way to do this.
```